manifests/config.pp

```diff
@@ -25,6 +25,13 @@
     content => template('foreman/database.yml.erb'),
   }
 
+  file { '/etc/systemd/system/httpd.service/10-longer_timeout.conf':
```

Comment:
I need to add an ensure => file here

Reply:
```ensure => file``` is indeed cleaner but is implied when you supply ```content```.